File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSExternDataDeclarationImpl.java
package net.kenro.ji.jin.purescript.psi.impl;

import com.intellij.lang.ASTNode;
import net.kenro.ji.jin.purescript.psi.PSExternDataDeclaration;

public class PSExternDataDeclarationImpl extends PSPsiElement implements PSExternDataDeclaration {

    public PSExternDataDeclarationImpl(final ASTNode node) {
        super(node);
    }

}

File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSExternDataDeclarationImpl.kt
package net.kenro.ji.jin.purescript.psi.impl

import com.intellij.lang.ASTNode
import net.kenro.ji.jin.purescript.psi.PSExternDataDeclaration

class PSExternDataDeclarationImpl(node: ASTNode) : PSPsiElement(node),
    PSExternDataDeclaration